### DISCUSSION TITLE
D106 : Ignore specific class in Nested Class

### DISCUSSION THREAD

**Discussion by yoann9344** [astral-sh/ruff#10236] [Q&A]
Is there any possibilities to ignore specific class for the error https://docs.astral.sh/ruff/rules/undocumented-public-nested-class/ ?  
For example, I use a Meta all over my classes, and I don't need to repeat the documentation for it.
```python
class SomeClass(...):
    class Meta:  # noqa: D106 - Cancel the check for `Meta`
        ...
```

**Comment by MichaReiser** [comment]
No such configuration exists to my knowledge. You might want to disable it for your project if it gives you too many false positives.

**Reply by UnoYakshi** [reply]
`Meta` sub-classes are quite common for Django/DRF-based projects. Won't there be any fast workarounds to ignore class names (by wildcard) in the future?..

**Comment by skissane-medallia** [comment]
Note there is an issue open to add this feature: https://github.com/astral-sh/ruff/issues/11211
